### PR TITLE
CRM-16901: Offline Recurring Contributions - Revised

### PR DESCRIPTION
**Pull request by rajeshrhino**
Offline recurring contributions - Revised
- Features:
  - Recurring Contribution Summary report
  - Improvements to Recurring Contribution search (Advanced Search / Find Contributions
  - BAO/API support for custom fields on contribution_recur records
  - Improvements to Manual Payment Processor

---
- CRM-16901: Offline Recurring Payments
  https://issues.civicrm.org/jira/browse/CRM-16901

### CODE REVIEW

File: templates/CRM/Contribute/Form/Search/Common.tpl [hunk 1]
```diff
@@ -160,11 +160,6 @@
 {include file="CRM/Campaign/Form/addCampaignToComponent.tpl" campaignContext="componentSearch"
 campaignTrClass='' campaignTdClass=''}
 
-<tr>
-  <td colspan="2">
-{include file="CRM/Contribute/Form/Search/ContributionRecur.tpl"}
```
Comment (davecivicrm) [inline templates/CRM/Contribute/Form/Search/Common.tpl]: Removing this include seems wrong. I'm not seeing the recurring contribution section at all in Find Contributions and Advanced Search.

Comment (rajeshrhino) [inline templates/CRM/Contribute/Form/Search/Common.tpl]: The new tpl file was missed in the commit. I have committed the file now. 

Can you please try again.

**Comment by colemanw** [comment]
@rajeshrhino can you please rebase this branch?

**Comment by davecivicrm** [comment]
Closed in favor of https://github.com/civicrm/civicrm-core/pull/7295.

**Pull request closed without merging**